lib/cisco_node_utils/cmd_ref/interface_channel_group.yaml

```diff
@@ -16,18 +16,34 @@ channel_group:
   kind: int
   default_value: false
   nexus:
-    get_value: '/^channel-group (\d+)$/'
+    get_value: '/^channel-group (\d+)\s?\S*\s?\S*$/'
```

Comment:
The `$/` might be problematic for portchannels with a configured mode. It won't match with the end-of-line token. if it's excluded, it should not affect the existing `channel_group` prop.